Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:3.9-bullseye as base
 FROM base as builder
 RUN mkdir /install
 WORKDIR /install
-RUN pip install --prefix=/install ostorlab
+RUN pip install --prefix=/install ostorlab -U
```

Comment:
This should be built from the code, otherwise it will pick an older version from pypi.